core/src/com/po/fuck/model/Constants.java

```diff
@@ -1,51 +1,135 @@
+    public static final int WEAPON_LAYER = loadConstant("WEAPON_LAYER", Integer.class);
+    public static final int GUI_LAYER = loadConstant("GUI_LAYER", Integer.class);
 
     /* borders */
-    public static final int GAME_BORDER = 20000;
+    public static final int GAME_BORDER = loadConstant("GAME_BORDER", Integer.class);
+
+    /* bullets */
+    public static final float COSMIC_BULLET_LIFE_TIME = loadConstant("COSMIC_BULLET_LIFE_TIME", Float.class);
+    public static final float COSMIC_BULLET_DAMAGE = loadConstant("COSMIC_BULLET_DAMAGE", Float.class);
+    public static final float COSMIC_BULLET_SPEED = loadConstant("COSMIC_BULLET_SPEED", Float.class);
+    public static final float LASER_BEAM_DAMAGE = loadConstant("LASER_BEAM_DAMAGE", Float.class);
+    public static final float LASER_BEAM_LIFE_TIME = loadConstant("LASER_BEAM_LIFE_TIME", Float.class);
+    public static final float DEFAULT_BULLET_LIFE_TIME = loadConstant("DEFAULT_BULLET_LIFE_TIME", Float.class);
+
+    public static final float GLOCK_COOLDOWN = loadConstant("GLOCK_COOLDOWN", Float.class);
+    public static final float LASERGUN_COOLDOWN = loadConstant("LASERGUN_COOLDOWN", Float.class);
+
+    public static final float PLAYERS_HEALTH = loadConstant("PLAYERS_HEALTH", Float.class);
+
+    public static final float DEFAULT_DISTANCE_FACTOR = loadConstant("DEFAULT_DISTANCE_FACTOR", Float.class);
+    public static final float DEFAULT_MAX_DISTANCE_FROM_BODY = loadConstant("DEFAULT_MAX_DISTANCE_FROM_BODY", Float.class);
 
-    /* bullets */ /* TODO: move to .properties */
-	public static final float COSMIC_BULLET_LIFE_TIME = 10f;
-    public static final float COSMIC_BULLET_DAMAGE = 1f;
-    public static final float COSMIC_BULLET_SPEED = 600f;
-    public static final float LASER_BEAM_DAMAGE = 3f;
-    public static final float LASER_BEAM_LIFE_TIME = 1f;
-    public static final float DEFAULT_BULLET_LIFE_TIME = 60f;
+    public static final float BASIC_ENEMY_SPEED = loadConstant("BASIC_ENEMY_SPEED", Float.class);
+    public static final float BASIC_ENEMY_HEALTH = loadConstant("BASIC_ENEMY_HEALTH", Float.class);
+    public static final int BASIC_ENEMY_REWARD = loadConstant("BASIC_ENEMY_REWARD", Integer.class);
 
-    public static final float GLOCK_COOLDOWN = 0.5f;
-    public static final float LASERGUN_COOLDOWN = 1f;
+    public static void saveToProperties() {
+        String propertiesFilePath = "../core/src/com/po/fuck/model/.properties";
+    
+        // Ensure the file exists or create it if it doesn't
+        File propertiesFile = new File(propertiesFilePath);
+        if (!propertiesFile.exists()) {
+            try {
+                propertiesFile.getParentFile().mkdirs(); // Create parent directories if they don't exist
+                propertiesFile.createNewFile(); // Create the file
+            } catch (IOException e) {
+                throw new RuntimeException("Unable to create properties file: " + e.getMessage());
```

Comment:
> throw new RuntimeException("Unable to create properties file: " + e.getMessage());

Is there not enough information to catch them and rethrow ?

Reply:
It is a way to close an application with easier to debug information.